### PR TITLE
AN-1063 do not run loadEmptyDataGraph for each test

### PR DESCRIPTION
**Pull request by vkagamlyk**
*Issue #, if available:*
https://bitquill.atlassian.net/browse/AN-1063

*Description of changes:*
Investigate Gorilla transport layer issues

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: gremlin-go/cucumber/tinkerPopWorld.go [hunk 1]
```diff
@@ -115,6 +115,10 @@ func (t *TinkerPopWorld) loadAllDataGraph() {
 }
 
 func (t *TinkerPopWorld) loadEmptyDataGraph() {
+	if _, ok := t.graphDataMap["empty"]; ok {
```
Comment (xiazcy) [inline gremlin-go/cucumber/tinkerPopWorld.go]: The reconnect for empty graphs is actually achieve on `line 716` in `featureSteps_test.go`, where it calls `loadEmptyDataGraph()` before every scenario. If you want to share the connection, there is no need to add this check here, just delete that line instead.

**Review by xiazcy — CHANGES_REQUESTED** [review]
To reuse connection for empty graph, remove `line 716` in `featureSteps_test.go` instead.

File: gremlin-go/cucumber/tinkerPopWorld.go (hunk 1 above)
Comment (vkagamlyk) [reply gremlin-go/cucumber/tinkerPopWorld.go]: done in https://github.com/lyndonbauto/tinkerpop/pull/52/commits/e4a7ea2c909eb272e8eeab552a4c559f34471917

**Review by lyndonbauto — APPROVED** [review]
LGTM

**Review by xiazcy — APPROVED** [review]

**Pull request merged**